photomaker/model/model.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import torch
+from huggingface_hub import hf_hub_download
+from photomaker import PhotoMakerStableDiffusionXLPipeline
+from PIL import Image
+
+
+class PhotoMakerModel:
+    def __init__(self):
+        self.model = None
+        self.device = "cuda" if torch.cuda.is_available() else "cpu"
+
+    def load_model(self):
+        # Download the model from Hugging Face Hub
+        model_path = hf_hub_download(
+            repo_id="TencentARC/PhotoMaker",
```

Comment:
you should use `model_cache` instead of downloading the model here. Take a look at some of the other examples please.